src/deploy_tools/app_builder.py

```diff
@@ -96,3 +102,39 @@ def _create_shell_file(self, app: ShellApp, module: Module) -> None:
             executable=True,
             create_parents=True,
         )
+
+    ALL_READ_EXECUTE_PERMISSIONS = 0o555
```

Comment:
Module level constants are normally defined at the top of the file, just under the imports.

Reply:
moved